app/src/main/java/com/example/habitsmasher/ui/dashboard/AddHabitDialog.java

```diff
@@ -119,14 +128,15 @@ public void onClick(View view) {
                 String habitTitle = _habitTitleEditText.getText().toString();
                 String habitReason = _habitReasonEditText.getText().toString();
                 String habitDate = _habitDateTextView.getText().toString();
+                boolean habitPublic = publicPrivateButtons.isPublic();
```

Comment:
Minor: could be renamed to `isHabitPublic`

Reply:
👍